superpeer/handler/MenuHandler.py

```diff
@@ -22,11 +29,77 @@ def serve(self, choice: str) -> None:
 		elif choice == "QUER":
 			pass
 
+		elif choice == "LISTSUPERPEERS":
+
+			shell.print_green('\nList of known peers:')
+
```

Comment:
Aggiungiamo anche qua il messaggio in caso di nessunsuper peer conosciuto, che dici?

Reply:
aggiunto! ;)